mixer/pkg/protobuf/yaml/dynamic/valueTypeEncoder.go

```diff
@@ -92,6 +96,16 @@ func valueTypeEncoderBuilder(_ *descriptor.DescriptorProto, fd *descriptor.Field
 					vVal.Value = &v1beta1.Value_DoubleValue{ev}
 					return nil
 				}
+			case v1beta1.TIMESTAMP:
+				enc = func(bag attribute.Bag, vVal *v1beta1.Value) error {
+					ev, er := expr.Evaluate(bag)
+					if er != nil {
+						return er
+					}
+					ts := ev.(time.Time)
+					vVal.Value = &v1beta1.Value_TimestampValue{&v1beta1.TimeStamp{&types.Timestamp{ts.Unix(), int32(ts.Nanosecond())}}}
```

Comment:
Use https://godoc.org/github.com/gogo/protobuf/types#TimestampProto to do this instead.

Reply:
done.